src/main/java/org/informiz/model/SourceBase.java

```diff
@@ -70,6 +71,7 @@ public String getDisplayValue() {
     @NotNull(message = "Type is mandatory", groups = {SourceFromUI.class, Default.class})
     private SourceType srcType;
 
+    @Size(max = 500, message = "Description must be under 500 characters", groups = {SourceBase.SourceFromUI.class, Default.class})
```

Comment:
Add "TODO" comment to match size restrictions to the ones defined in the database